prometheus/create-datasource-and-dashboard.sh

```diff
@@ -11,8 +11,10 @@ fi
 dashboard_id=1598
 last_revision=$(curl -sf https://grafana.com/api/dashboards/${dashboard_id}/revisions | grep '"revision":' | sed 's/ *"revision": \([0-9]*\),/\1/' | sort -n | tail -1)
 
-curl -s https://grafana.com/api/dashboards/${dashboard_id}/revisions/${last_revision}/download | \
-    xargs -0 -I "{}" curl --retry-connrefused --retry 5 --retry-delay 0 -sf \
-          -X POST -H "Content-Type: application/json" \
-          --data-binary '{"dashboard": {}, "inputs": [{"name": "DS_PROM", "pluginId": "prometheus", "type": "datasource", "value": "prom"}], "overwrite": false}' \
-          http://grafana:3000/api/dashboards/import
+echo '{"dashboard": ' > data.json
+curl -s https://grafana.com/api/dashboards/${dashboard_id}/revisions/${last_revision}/download >> data.json
+echo ', "inputs": [{"name": "DS_PROM", "pluginId": "prometheus", "type": "datasource", "value": "prom"}], "overwrite": false}' >> data.json
```

Comment:
This rewrite was needed because as the dashboard JSON grows (more graphs added), we ran into this issue:

```
xargs: argument line too long
```